assets/less/icons.less

```diff
@@ -18,21 +18,23 @@
 /* had to add a custom template to the generator to get this to compile to append the proper path and specify  "url" instead of "font-url":
 https://github.com/seanbarclay/fontcustom/blob/1c3ad3d2633d3a8add39e8827115394e9ec02d88/lib/fontcustom/generator/template.rb
 */
+@fauxtonFont: "fauxtonicon1";
+
 @font-face {
-  font-family: "fauxtonicon";
-  src: url("../fonts/fauxtonicon.eot");
-  src: url("../fonts/fauxtonicon.eot?#iefix") format("embedded-opentype"),
-       url("../fonts/fauxtonicon.woff") format("woff"),
-       url("../fonts/fauxtonicon.ttf") format("truetype"),
-       url("../fonts/fauxtonicon.svg#fauxtonicon") format("svg");
+  font-family: "fauxtonicon1";
+  src: url("../fonts/fauxtonicon1.eot");
+  src: url("../fonts/fauxtonicon1.eot?#iefix") format("embedded-opentype"),
+       url("../fonts/fauxtonicon1.woff") format("woff"),
+       url("../fonts/fauxtonicon1.ttf") format("truetype"),
+       url("../fonts/fauxtonicon1.svg#fauxtonicon1") format("svg");
```

Comment:
Actually it is! It's stored in the yaml file in the same folder. `font_name` or something: this is the generated less content.